ros/node.go

```diff
@@ -149,8 +150,8 @@ func newDefaultNode(name string, args []string) (*defaultNode, error) {
 
 	node.nameResolver = newNameResolver(node.namespace, node.name, remapping)
 	node.nonRosArgs = rest
-
-	node.qualifiedName = node.namespace + "/" + node.name
+	// Removed additional "/" from node.qualifiedName which breaks rosgraph tests in roswtf
```

Comment:
Don't put comments in which just explain how this version was changed; that's just source control meta-data.